docs/TablePartitionsStorageGroups.md

```diff
@@ -0,0 +1,421 @@
+        - [Automatic archive initial import](#automatic-archive-initial-import)
+    - [Troubleshooting](#troubleshooting)
+        - [Storage Group Assignment Mismatch](#storage-group-assignment-mismatch)
+        - [Backup Failure](#backup-failure)
+        - [Storage Group Startup Failure](#storage-group-startup-failure)
+
+<!-- /TOC -->
+</details>
+
+## Introduction
+
+Table partitioning is the mechanism by which a version of a database table row is stored on a subset of Storage Managers (SMs) running in a NuoDB database.
+A table partition is an attribute that can be associated with a database table.
+A storage group represents a logical storage unit that can be serviced by multiple SMs in a database.
+Each partition within a table has a uniquely identified name that maps to one storage group (SG) which represents a logical storage unit that can be serviced by multiple SMs in a database.
+
+This document explains how to configure Table Partitions and Storage Groups (TPSG) in NuoDB Kubernetes deployments.
+
+>**Note**
+> For non-Kubernetes deployments, see  [Table Partitions and Storage Groups][2]. This page expands on the product documentation and is specific to this Helm Chart repository.
+
+## Deployment Overview
+
+NuoDB [database](../stable/database) Helm chart can be installed multiple times in a single Kubernetes namespace where there is one _primary_ and one or more _secondary_ Helm release.
+A storage group is deployed as a separate _secondary_ Helm release which also deploys the SMs servicing this storage group.
+This allows the user to remove any storage group without removing resources installed by the _primary_ database Helm release.
+To simplify TPSG operations and observability, NuoDB Helm charts support one storage group per Storage Manager.
+
+For information on adding storage groups, see [Managing Storage Groups](#managing-storage-groups).
+
+### Prerequisites
+
+- NuoDB 5.0.4 or later
+- NuoDB Helm charts 3.6.0 or later
+
+### Example
+
+The below diagram shows an example of a TPSG deployment in a single Kubernetes cluster with two storage groups.
+
+![TPSG Example Deployment](../images/TPSG-deployment.png)
+
+Before continuing, make sure that you have a running NuoDB Administration layer in the target namespace.
+For more information, see [NuoDB Admin Helm Chart](../stable/admin/).
+
+In this example, a NuoDB database with two storage groups is provisioned into the Kubernetes namespace.
+The Transaction Engines (TEs) are part of the _primary_ Helm release which is deployed first.
+One backup group is configured that includes all Hot-copy Storage Managers (HCSMs) as there will be one deployed per storage group.
+For more information on configuring backup when TPSG is enabled, see [Backup Database with TPSG](#backup-database-with-tpsg).
+
+```sh
+helm upgrade --install database stable/database \
+  --set database.te.replicas=3 \
+  --set database.sm.hotCopy.replicas=0 \
+  --set database.sm.noHotCopy.enablePod=false \
```

Comment:
Is the difference between `database.sm.noHotCopy.enablePod=false` and `database.sm.noHotCopy.replicas=0` that the former suppresses the backup cronjob? 

It might be clearer and more complete to specify `database.sm.noHotCopy.replicas=0` even if it is redundant with `enablePod=false`, since it shows all the workload replica counts.

Reply:
The `database.sm.hotCopy.enablePod=false` also suppresses the CronJob rendering which is unfortunate as there is another variable (`database.sm.hotCopy.enableBackups`) that controls this. We need the CronJobs here, hence, `database.sm.hotCopy.replicas=0` is set. The downside is that the HCSM StatefulSet will also be rendered but it's not a problem.

I will change `database.sm.noHotCopy.enablePod=false` -> `database.sm.noHotCopy.replicas=0` in the primary release so that it's consistent.